pregred-gui/src/app/components/app-footer/AppFooter.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './AppFooter.css';
+import {
+  FacebookOutlined,
+  InstagramOutlined,
+  YoutubeOutlined,
+  TwitterOutlined
+} from '@ant-design/icons';
+
+
+
+
+function AppFooter() {
+    return(
+      <div className="footer-container">
+        <div className="f-text">
+          <ul className="dot">
+            <li>About Us: <a href="www.pregred.com"> www.pregred.com </a> </li>
+            <li>Contact Us: 0768660203</li>
```

Comment:
As i know, About Us and Contact Us  are going to navigate to respective pages. so, 'www.pregred.com' and mobile numbers are not needed.